docs/usage/billing.md

```diff
@@ -45,151 +33,40 @@ This setting is a collection of billing plans. Each billing plan allows the foll
-  const sessionId = shopify.session.getCurrentId({
-    isOnline: true,
-    rawRequest: req,
-    rawResponse: res,
-  });
-
-  // use sessionId to retrieve session from app's session storage
-  // getSessionFromStorage() must be provided by application
-  const session = await getSessionFromStorage(sessionId);
-
-  const confirmationUrl = await shopify.billing.request({
-    session,
-    // Receive the selected plan from the frontend
-    plan: req.body.selectedPlan,
-    isTest: true,
-  });
-
-  res.redirect(confirmationUrl);
-});
-```
-
-The `request` method accepts the following parameters:
-
-| Parameter | Type      | Required? | Default Value | Notes                                                |
-| --------- | --------- | :-------: | :-----------: | ---------------------------------------------------- |
-| `session` | `Session` |    Yes    |       -       | The `Session` for the current request                |
-| `plan`    | `string`  |    Yes    |       -       | Which plan to create a charge for                    |
-| `isTest`  | `boolean` |    No     |    `true`     | If `true`, Shopify will not charge for this purchase |
-
-It will return a `string` containing a URL - we don't redirect right away to make it possible for apps to run their own code after it creates the payment request.
-
-The app **must** redirect the merchant to this URL so that they can confirm the charge before Shopify applies it.
-The merchant will be sent back to your app's main page after the process is complete.
-
 ## When should the app check for payment?
 
 As mentioned above, billing requires a session to access the API, which means that the app must actually be installed before it can request payment.
 
 With the `check` method, your app can block access to specific endpoints, or to the app as a whole.
 If you're gating access to the entire app, you should check for billing:
 
-1. After OAuth completes, you'll get the session back from `shopify.auth.callback`. You can use the session to ensure billing takes place as part of the authentication flow.
-1. When validating requests from the frontend. Since the check requires API access, you can only run it in requests that work with `shopify.session.getCurrentId`.
+1. After OAuth completes, you'll get the session back from [`shopify.auth.callback`](../reference/auth/callback.md). You can use the session to ensure billing takes place as part of the authentication flow.
+1. When validating requests from the frontend. Since the check requires API access, you can only run it in requests that work with [`shopify.session.getCurrentId`](../reference/session/getCurrentId.md).
 
 **Note**: the merchant may refuse payment when prompted or cancel subscriptions later on, but the app will already be installed at that point. We recommend using [billing webhooks](https://shopify.dev/apps/billing#webhooks-for-billing) to revoke access for merchants when they cancel / decline payment.
 
```

Comment:
Similar to the reference comment, really like the link back to guide index here. Could we add to the other guides?